core/node/groups.go

```diff
@@ -264,25 +226,27 @@ func Online(bcfg *BuildCfg, cfg *config.Config) fx.Option {
 	}
 
 	return fx.Options(
-		fx.Provide(OnlineExchange),
+		fx.Provide(OnlineExchange(!cfg.Experimental.StrategicProviding)),
```

Comment:
Can we comment like `/* don't provide form bitswap when the strategic provider service is active */`?

Reply:
I used a variable with a more descriptive name and added the comment.